src/btree.c

```diff
@@ -2552,7 +2552,8 @@ btree_iterator_print(iterator *itor)
    btree_print_node(Platform_default_log_handle,
                     btree_itor->cc,
                     btree_itor->cfg,
-                    &btree_itor->curr);
+                    &btree_itor->curr,
+                    PAGE_TYPE_BRANCH);
```

Comment:
I suppose BTree iterators could span both memtable and branch btrees. I had to make a choice, and went with this one. 

Not sure if this has to be made more 'clever' to figure out the page's type based on which sub-tree the iterator is working upon.